src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -399,4 +400,23 @@ public function iClickNearInVariant($button, $value)
         $tr = $this->assertSession()->elementExists('css', sprintf('table#variants tbody tr:contains("%s")', $value));
         $tr->clickLink($button);
     }
+
+    /**
+     * @Given /^I fill in "([^"]*)" translation "([^"]*)" with "([^"]*)"$/
+     */
+    public function iFillInTranslationWith($label, $locale, $value)
+    {
+        /** @var NodeElement[] $inputs */
+        $inputs = $this->getSession()->getPage()->findAll('css', sprintf('label:contains("%s") ~ div input', $label));
+
+        foreach ($inputs as $input) {
+            $id = $input->getAttribute('id');
+            if(false !== stripos($id, $locale)){
+                return $input->setValue($value);
+            }
+        }
+
+        throw new \Exception('No locale translation!');
```

Comment:
I'm not sure about message of this exception, but I have no idea how should it look like :smile: